config/rbac/storedebuginstance_admin_role.yaml

```diff
@@ -0,0 +1,27 @@
+# This rule is not used by the project shopware-operator itself.
+# It is provided to allow the cluster admin to help manage permissions for users.
+#
+# Grants full permissions ('*') over shop.shopware.com.
+# This role is intended for users authorized to modify roles and bindings within the cluster,
+# enabling them to delegate specific permissions to other users or groups as needed.
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
You don't need the clusterRole for this, because each operator only is allowed to work in his own namespace. You can change this in the controler file and set a 'namespace=default' so the controler will generate only a Role